traits/has_traits.py

```diff
@@ -305,6 +306,16 @@ def _property_method(class_dict, name):
     return class_dict.get(name)
 
 
+def _mapped_trait_default_method(trait, trait_name):
+
```

Comment:
Please could we have a docstring here? (Even though it's a private function, it'll still help people who are reading and understanding the code.)